docs/src/quickstart.md

```diff
@@ -49,10 +54,8 @@ end
 !!! note
     Your model doesn't have to be called `model` - it's just a name.
 
-There are a few options for defining a variable, depending on whether you want
-to have lower bounds, upper bounds, both bounds, or even no bounds. The
-following commands will create two variables, `x` and `y`, with both lower and
-upper bounds. Note the first argument is our model variable ``model``. These
+The following commands will create two variables, `x` and `y`, with both lower
+and upper bounds. Note the first argument is our model variable `model`. These
```

Comment:
"is our model `model`". Calling the model a "variable" is confusing in the next sentence